### DISCUSSION TITLE
Custom _id for MongoDB

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#11784] [Q&A] [answered]
Is it possible to have arbitrary values for "_id" in MongoDB? Like strings or numbers? I am checking the documentation but no clue about how to do it.

https://www.prisma.io/docs/concepts/database-connectors/mongodb#using-objectid

I am trying to use Prisma Studio but I always get an error when I try to define an arbitrary value because is not compatible with "ObjectID" MongoDB specifications.

**Comment by janpio** [comment, marked as answer]
Yes, it should be possible if you configure the schema correctly.

Can you share what your Schema Prisma looks like when you get the error message (and also include the error message)?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
